.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,8 +4,9 @@ Developers are strongly encouraged to first make a PR to their own plotly.js for
 
 Before opening a pull request, developer should: 
 
-- `git rebase` their local branch off the latest `master`
-- make sure to **not** `git add` the `dist/` folder (the `dist/` is updated only on verion bumps)
-- write an overview of what the PR attempts to do.
+- `git rebase` their local branch off the latest `master`,
+- make sure to **not** `git add` the `dist/` folder (the `dist/` is updated only on verion bumps),
+- write an overview of what the PR attempts to do,
+- select the _Allow edits from maintainers_ option (see this [article](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) for more details).
```

Comment:
from https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/